### ISSUE TITLE
Nested arrays produce invalid binary data error

### ISSUE THREAD

**Issue by u5er1ame** [surrealdb/surrealist#370]
**Describe the bug**
Can't add record to table because connection drops immediately
If data already in the table binary data error appears selecting whole table with query or in the explorer
**To Reproduce**
try add following record to table
```
{ geometry: { type: 'Polygon', coordinates: [[[36.77269829083, 55.66466213383], [36.77269384177, 55.6513182831], [36.8068593987, 55.65131451768], [36.8068593987, 55.66466213383], [36.77269829083, 55.66466213383]]] }, properties: {}, type: 'Feature' }
```
connection drops (some warning appears in record editor but it hides)
its totally valid data you can add with cli

If you add it with cli and then select it with `SELECT * FROM table` or `SELECT geometry FROM table` in surrealist
even then select table in explorer binary data error appears

No errors in surreal cli

I assume something wrong with nested array parsing because selecting individual fields or even  `SELECT VALUE geometry.coordinates FROM table` works fine
**Expected behavior**
record added to table
select works properly 

**Environment:**
```
Version: 2.1.5
Flags: feature_flags: false, models_view: true, apidocs_view: true, themes: false, newsfeed: true, database_version_check: true, highlight_tool: false, surreal_compat: v1, changelog: auto

```

**Comment by kearfy** [comment]
Hey @u5er1ame, this is an issue with polygons. It should be solved once 2.0.0 is released

**Comment by macjuul** [comment]
This will be fixed in the next SurrealDB 2.0 compatible release 🙂
